docs/visualize/lens.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[lens]]
+== Lens
+
+beta[]
+
+*Lens* provides you with a simple way to create basic visualizations.
+
+With Lens, you can:
+
+* Easily make sense of your data with basic visualizations.
+* Use aggregations to plot any data set, no matter how large.
+* Focus the data you want to display with searches.
+* Add your visualizations to Dashboards and Discover.
+
+//TODO For an overview of Lens, watch <<Lens video, Getting started with Lens>>.
+
+[float]
+[[choose-lens-visualization]]
+=== Types of Lens visualizations
+
+Lens comes with these types of visualizations:
```

Comment:
I like this section, but I think it takes up too much space.  What about using thumbnails for the images? And maybe placing the visualizations two across?

Reply:
I can make the images smaller, but tables go haywire when introduced to images.